tools/operator/testdata/12HrLongRun/samplecc-input.yml

```diff
@@ -0,0 +1,90 @@
+tls: true
```

Comment:
Let's rename this file to be a more functional name, and move it put up a directory level, so anyone can find it. No reason to scope it inside a test directory. e.g.: 
    testdata/12hr80tps4org2chan-test-input.yml

Reply:
I will change the file name. But why are we specifying 80tps in the file name?

Reply:
I guess the tps number is not as important - but it is the input rate. Most of our tests threads just send as much as possible, bit this one restricts input rate to 10 TX per sec on each thread. That is useful to know.

Reply:
okay